tests/test_models.py

```diff
@@ -887,6 +887,17 @@ def test_predict_h(self):
         scores = self.instance.predict_h(rt_batch=rt_batch)
         assert torch.allclose(scores, expected_scores)
 
+    def test_predict_r(self):
+        """Test predict_r."""
+        ht_batch = self._combination_batch(relations=False)
+        expected_scores = self.instance._generate_fake_scores(
+            h=ht_batch[:, 0, None],
+            r=torch.arange(self.factory.num_relations).unsqueeze(dim=0),
```

Comment:
```suggestion
            r=torch.arange(self.factory.num_relations, device=self.instance.device).unsqueeze(dim=0),
```

Depending on the question above (what should `score_r` actually return) this should be `num_real_relations` instead.

Reply:
Now that #726 is done, is the answer to this clear?